NEXT_CHANGELOG.md

```diff
@@ -60,30 +60,11 @@ By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographq
 
 ## 🚀 Features ( :rocket: )
 
-### Add support of multiple uplink URLs [PR #1210](https://github.com/apollographql/router/pull/1210)
```

Comment:
these should not be removed from the next_changelog, right? Some entries are from main, some from api-1.0